appveyor.yml

```diff
@@ -76,6 +76,9 @@
       - nodejs_version: 12
         GYP_MSVS_VERSION: 2017
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      - nodejs_version: "13.0.0"
+        GYP_MSVS_VERSION: 2017
+        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
```

Comment:
There's another environment matrix below too. It was updated in the commit for the Node v12 release. https://github.com/sass/node-sass/commit/da10866c45d2c65ece1bed260af3e99423aead34#diff-180360612c6b8c4ed830919bbb4dd459

Reply:
yeah, right, thanks